lib/smart_answer_flows/marriage-abroad/outcomes/outcome_os_commonwealth.govspeak.erb

```diff
@@ -43,11 +43,36 @@
     <% end %>
   <% when 'india' %>
 
-    The British High Commission in India can provide you with a letter explaining this position for a charge - see [consular fees for India](/government/publications/india-consular-fees).
+    Download and print the [information note on ‘no objection certificates’](https://www.gov.uk/government/publications/marriage-information-note-on-noc) and show it to the registrar.
```

Comment:
Change made in this [commit](https://github.com/alphagov/smart-answers/pull/2365/commits/5037c0f38a4fc1cd76f25849e99e5938d3209c1a)